README.md

```diff
@@ -50,12 +50,6 @@ yarn dev
 |ExampleFetchComponent.tsx|AppengGHReposComponent.tsx|
 |ExampleFetchComponent.test.tsx|AppengGHReposComponent.test.tsx|
 
```

Comment:
Hi @yzhao583 Thanks. We may need to keep this section as the section with title `Ensure AppengGHReposComponent is now being used` makes use of this to prove the updates have taken place.

Reply:
@rhkp I just cloned the repo and followed the README, and went to the "AppengGHReposComponent.tsx" file, and I did not find any email, so I felt confused. In addition, the code in "ExampleFetchComponent.tsx" is just trying to simulate a fetch operation, and it is already replaced by a real API call in "AppengGHReposComponent.tsx", so I think it is not necessary to still keep the related content there. WDYT?

Reply:
Okay then let us remove this section.